sdk/keyvault/keyvault-admin/tests.yml

```diff
@@ -1,25 +1,23 @@
 trigger: none
 
-extends:
-  template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
-  parameters:
-    PackageName: "@azure/keyvault-admin"
-    ResourceServiceDirectory: keyvault
-    # KV HSM limitation prevents us from running live tests
-    # against multiple platforms in parallel (we're limited to a single
-    # instance per region per subscription) so we're only running
-    # live tests against a single instance.
-    TestMinMax: false
-    TestBrowser: false
-    testSamples: false
-    ArmTemplateParameters: "@{ enableHsm = $true }"
-    Matrix:
-      Linux Node 10 with Managed HSM:
-        Pool: azsdk-pool-mms-ubuntu-1804-general
-        OSVmImage: MMSUbuntu18.04
-        TestType: "node"
-        NodeTestVersion: "10.x"
-    EnvVars:
-      AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-      AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-      AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/keyvault-admin"
+      ServiceDirectory: keyvault
+      # KV HSM limitation prevents us from running live tests
+      # against multiple platforms in parallel (we're limited to a single
+      # instance per region per subscription) so we're only running
+      # live tests against a single instance.
+      Location: eastus2
+      MatrixFilters:
+        - TestType=node
+      MatrixConfigs:
```

Comment:
This is great, thanks so much! For KV Admin, it currently only works on MHSM so we only need to run it against MHSM; however, in the future (and today for KV Keys) we will need something like:

Generate the default matrix + an additional config where I pass `enableHsm = $true`... how would I define that?

Reply:
Oh I should have kept reading... AdditionalMatrixConfigs is my friend there I assume!

Reply:
Take a look at what I'm doing with the storage and attestation tests, it's the same scenario: https://github.com/Azure/azure-sdk-for-js/blob/b099113c75b05a06865cc344eaed03e45111e47b/sdk/storage/storage-blob/platform-matrix.json